common.xml

```diff
@@ -129,4 +128,17 @@
 		<taskdef name="junitlauncher" classname="org.apache.tools.ant.taskdefs.optional.junitlauncher.confined.JUnitLauncherTask"
 			classpath="${build.dir}/ant-junitlauncher.jar"/>
 	</target>
+
+
+    <macrodef name="gpgsignfile">
+        <attribute name="target"/>
+        <sequential>            
+            <exec executable="gpg" dir="${build.release.dir}" failonerror="true">
+                <arg value="-ab"/>
```

Comment:
how is the GPG private key protected?

Reply:
It's expected to be in the user's directory. Unfortunately there's no way to for our build to enforce a developer having protections. 

The next step would be to add a KEYS file to the repository and provide instructions to those who would want to verify.